WooCommerce/Classes/ViewRelated/Coupons/UsageDetails/CouponUsageDetailsViewModel.swift

```diff
@@ -0,0 +1,72 @@
+import Combine
+import Foundation
+import Yosemite
+
+/// View Model for `CouponUsageDetails`
+///
+final class CouponUsageDetailsViewModel: ObservableObject {
+    private let coupon: Coupon
+    private let currencySettings: CurrencySettings
+
+    var currencySymbol: String {
+        currencySettings.symbol(from: currencySettings.currencyCode)
```

Comment:
nit: not sure if it's worth setting this in init so that this value stays the same?